docker-compose.yml

```diff
@@ -5,4 +5,19 @@ services:
     image: php-server
```

Comment:
php-server should eventually depend on mongodb so i think you should add that to the php-server service